multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/helpers/MtaDescriptorPropertiesResolver.java

```diff
@@ -147,6 +171,13 @@ private String replacePartsWithIdlePlaceholders(String uriString, boolean noHost
         return uri.toString();
     }
 
+    private String replacePartsWithLivePlaceholders(String uriString, boolean noHostname, String protocol) {
```

Comment:
maybe reuse code by extracting replacePartsWithIdlePlaceholders( method in a more generic 'replaceUriPartsWithPlaceholders, that is called for both Idle and Live placeholders.